.config/nextest.toml

```diff
@@ -4,7 +4,7 @@ test-threads = "num-cpus"
 
 [[profile.default.overrides]]
 # Remove once https://github.com/thin-edge/thin-edge.io/issues/3021 is resolved
-filter = 'test(bridge_reconnects_successfully_after_local_connection_interrupted) or test(bridge_reconnects_successfully_after_cloud_connection_interrupted)'
```

Comment:
When has been these two flaky tests resolved? If so it would be good to close https://github.com/thin-edge/thin-edge.io/issues/3021.

Reply:
I'm not entirely sure, but I suspect it was with the reliability improvements you made in #3122. I've not seen the flakiness locally myself in cargo-nextest and I have seen quite a lot of flaky tests in recent weeks, I believe due to the strain it puts my work laptop under.